### ISSUE TITLE
Actually Additions Fishing Net

### ISSUE THREAD

**Issue by MrBirk** [Darkosto/SevTech-Ages#1959]
I made some actually addition Fishing Nets and setup with a chest and they haven't caught any fish. I saw that the Industrial foregoing fisher is also unable to catch fish. Is actually additions in the same boat?

Is the only way to catch fish with a rod by hand or with a strainer.

It seems kind of silly at age 4/5 to be unable to catch fish via a different method than manually or having to collect bait and setup fish traps.

**Comment by Targren** [comment]
Yeah, they probably both use the fishing loot tables, which are empty.

**Comment by artdude543** [comment]
@Darkosto I guess this will need to be tweaked? I remember you talking about this before but can't remember what you wanted to be done with it.

**Comment by justinrusso** [comment]
This will be fixed with an update to SevTweaks. This will have to wait until 3.0.9 due to some required updates that won't be in 3.0.8
